sdk/typescript/runtime/main.go

```diff
@@ -9,11 +9,30 @@ import (
 	"slices"
 	"strings"
 
-	"github.com/iancoleman/strcase"
+	"github.com/ettle/strcase"
 	"github.com/tidwall/gjson"
 	"golang.org/x/mod/semver"
 )
 
+var caser *strcase.Caser
+
+func init() {
+	caser = newCaser()
+}
+
+func newCaser() *strcase.Caser {
+	var splitFn = strcase.NewSplitFn(
+		[]rune{'*', '.', ',', '-', '_'},
+		strcase.SplitCase,
+		strcase.SplitAcronym,
+		strcase.PreserveNumberFormatting,
+		strcase.SplitBeforeNumber,
+		strcase.SplitAfterNumber,
+	)
+
+	return strcase.NewCaser(false, nil, splitFn)
```

Comment:
This is done this way to avoid dependency on dagql in runtime. But can we add this to `internal` package and use it from there? what do you think @jedevc @marcosnils?

Reply:
> Although, for sdk-runtime, it seems like there is no dependency on dagger's core/dagql package, so right now we are duplicating the code from strcase pkg into those files. But this mean, they can go out of sync quickly.

Yeah, the duplication is definitely not good :cry: I like the idea of adding a dependency onto the root module, that sounds fine to me - we can then have a `replace` directive in `elixir`/`typescript`/`python` to point it back to the top (though *maybe* this might require a bit of a chance to how the runtime gets built).

Another *potential* option would be to add this into the auto-generated `internal` package - but I'm not a huge fan of this - since it's kind of a work-around that only we're able to do - we should try and dogfood the other approach if we can.

Reply:
I actually like this idea. this way we may be able to omit the manual copy step. but I am not entirely sure if there are any other downsides to it.